brave/browser-laptop
# Sync: Update appState with Sync bookmark records

Test Plan:
1. Update `brave/sync`
2. Copy {User data}/brave-development to {User data}/brave-development-2
3. Reduce Sync interval to 10s `js/constants/appConfig.js` fetchInterval
4. Run brave the usual way (`yarn start` / `npm start`)>
5. Run another instance of brave with user-data-dir set to {Userdata}/brave-development-2. I did this by copying the package.json `scripts.start"` and changing user-data-dir on the copy.
6. Surf the Web and bookmark sites. Observe bookmarks sync.
7. Edit bookmark titles. Delete them. Make folders and move bookmarks into them.
